stash.py

```diff
@@ -23,3 +24,78 @@ def connect_firestore_with_key(collection_name: str, firestore_key: str):
         logger.info(f"{LOGGER_PREFIX}Firebase's firestore app is successfully connected")
         document_reference = database.collection(collection_name)
         return document_reference
+
+class HttpStatusCode(Enum):
+
+    # INFORMATIONAL RESPONSES (100–199)
+    CONTINUE = 100
```

Comment:
Each parameter of this enum class should have a tuple value something like _CONTINUE = (100, 'Information Response - Continue')_ .